google-api-client.gemspec

```
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'autoparse', '~> 0.3'
   s.add_runtime_dependency 'extlib', '~> 0.9'
   s.add_runtime_dependency 'launchy', '~> 2.4'
-  s.add_runtime_dependency 'retriable', '~> 1.4'
+  s.add_runtime_dependency 'retriable', '~> 2.0'
```

I think we will need to communicate an affected area to be tested Sign-in using google

---

nothing affected i guess
no code change except passing the interval in a different arguments, [ref](https://github.com/Instabug/google-api-ruby-client/blob/master/lib/google/apis/core/http_command.rb#L103-L104)

---

BTW i ran [these](https://github.com/Instabug/google-api-ruby-client/blob/v0.8-main/spec/google/api_client_spec.rb#L239-L290) specs and they passed